lib/core/device.js

```diff
@@ -232,11 +232,8 @@ device.detectUsb = function(driverId, cb) {
     vendorId: driverManifest.vendorId,
     productId: driverManifest.productId
   };
-  chrome.serial.getDevices(function(results) {
```

Comment:
Why remove logging of the connected devices. Doesn't that having that help us when things go wrong?